internal/corazawaf/waf.go

```diff
@@ -100,6 +100,7 @@ type WAF struct {
 	// UploadDir is the directory where the uploaded files will be stored
 	UploadDir string
 
+	// Request body in memory limit excluding the size of any files being transported in the request.
 	RequestBodyNoFilesLimit int64
```

Comment:
Actually, I think that the value is parsed and stored, but then never used. It should be implemented or documented (also in the [coraza.conf-recommended](https://github.com/corazawaf/coraza/blob/v3/dev/coraza.conf-recommended))